File: src/Java/conditionalStatements.java
package Java;

public class conditionalStatements {
    public static void main(String[] args) {
        //if else
        int a = 9;
        int b = 80;
        System.out.println(a > b);
        //switch case
        int age = 9;
        switch (age) {
            case 18:
                System.out.println("mature");
                break;
            case 25:
                System.out.println("extra mature");
                break;
            case 10:
                System.out.println("kid");
                break;
            default:
                System.out.println("ps go home");
        }
    }
}

File: src/Kotlin/conditionalStatements.kt
package Kotlin

fun main(arr: Array<String>) {
    //if else
    val a = 9
    val b = 80
    if (a > b) println(true) else println(false)

//if else as an expression
    var greater = if (a > b) a else b
    println(greater)

// when condition
    var age=89;
    when (age) {
        18 ->{ println("mature")}
        25 -> println("extra mature")
        10 -> println("kid")
        else -> println("ps go home")
    }

    //when as expresion
    var status=when (age) {
        18 ->"mature"
        25 -> "extra mature"
        10 -> "kid"
        else ->"ps go home"
    }
    println("status is $status ")

}